lib/rspec/core/output_wrapper.rb

```diff
@@ -0,0 +1,26 @@
+module RSpec
+  module Core
+    # @private
+    class OutputWrapper
+      # @private
+      attr_writer :output
+
+      # @private
+      def initialize(output)
+        @output = output
+      end
+
+      # Redirect calls for IO interface methods
+      (IO.instance_methods - Enumerable.instance_methods - Object.instance_methods + [:to_str]).each do |method|
```

Comment:
Can you just use `IO.instance_methods(false)` instead of subtracting the `Enumerable` and `Object` methods?

Also, what's special about `to_str` that requires it to be added here?  It's not clear why it should be delegated...